NuGetPackageVerifier.json

```diff
@@ -0,0 +1,127 @@
+{
+    "adx": { // Packages written by the ADX team and that ship on NuGet.org
+        "rules": [
+            "AssemblyHasDocumentFileRule",
+            "AssemblyHasVersionAttributesRule",
+            "AssemblyHasServicingAttributeRule",
+            "SatellitePackageRule",
+            "StrictSemanticVersionValidationRule"
+        ],
+        "packages": {
+            "dnx-clr-win-x64": {
+                "VERSION_INFORMATIONALVERSION": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this"
+                },
+                "SERVICING_ATTRIBUTE": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\Microsoft.CodeAnalysis.CSharp.dll": "Need to follow up, not from ADX team",
+                    "bin\\Microsoft.CodeAnalysis.dll": "Need to follow up, not from ADX team",
+                    "bin\\Microsoft.Dnx.Runtime.Internals.dll": "BUG: This file shouldn't be here"
```

Comment:
Just a comment in a PR I think https://github.com/aspnet/dnx/pull/2749#discussion_r39922395